projects/packages/my-jetpack/_inc/components/stats-section/index.jsx

```diff
@@ -11,7 +11,7 @@ const StatsSection = () => {
 	const slug = 'stats';
 	const { detail } = useProduct( slug );
 	const { status } = detail;
-	const { userIsAdmin } = window?.myJetpackInitialState ?? false;
+	const isAdmin = !! window?.myJetpackInitialState?.userIsAdmin ?? false;
```

Comment:
This was also causing a console error

Reply:
Oh wow, that previous solution was complicated 😅 I think the code is valid even when we remove ternary operation
```suggestion
	const isAdmin = !! window?.myJetpackInitialState?.userIsAdmin;
```

Reply:
You are correct, thanks!